virtcontainers/kata_agent_test.go

```diff
@@ -408,6 +408,62 @@ func TestHandleLocalStorage(t *testing.T) {
 	assert.Equal(t, localMountPoint, expected)
 }
 
+func TestHandleHugepages(t *testing.T) {
+	if os.Getuid() != 0 {
+		t.Skip("Test disabled as requires root user")
+	}
+
+	dir, err := ioutil.TempDir("", "hugepages-test")
+	assert.Nil(t, err)
```

Comment:
Nit: You can avoid passing the `t` param (easier to read ;-) if you use the following idiom:

```go
assert := assert.New(t)

// ...

assert.NoError(err)
```